test/integration/connectors/test_delta_table.py

```diff
@@ -0,0 +1,138 @@
+import multiprocessing
+import os
+from pathlib import Path
+
+import pytest
+from deltalake import DeltaTable
+from fsspec import get_filesystem_class
+
+from test.integration.connectors.utils.constants import (
+    DESTINATION_TAG,
+)
+from test.integration.utils import requires_env
+from unstructured_ingest.v2.interfaces import FileData, SourceIdentifiers
+from unstructured_ingest.v2.processes.connectors.delta_table import (
+    CONNECTOR_TYPE,
+    DeltaTableAccessConfig,
+    DeltaTableConnectionConfig,
+    DeltaTableUploader,
+    DeltaTableUploaderConfig,
+    DeltaTableUploadStager,
+    DeltaTableUploadStagerConfig,
+)
+
+multiprocessing.set_start_method("spawn")
```

Comment:
This is necessary for the tests to run asynchronously. The Delta table connector itself creates a process when uploading and without this line the second test will always fail.